include/constants/songs.h

```diff
@@ -1,64 +1,103 @@
-    SONG_0C = 0x0C,
-    SONG_0E = 0x0E,
-    SONG_0F = 0x0F,
-    SONG_BGM_MAP_PL10 = 0x010,
-    SONG_11 = 0x11,
-    SONG_12 = 0x12,
-    SONG_13 = 0x13,
-    SONG_14 = 0x14,
-    SONG_22 = 0x22,
-    SONG_23 = 0x23,
-    SONG_24 = 0x24,
-    SONG_25 = 0x25,
-    SONG_26 = 0x26,
-    SONG_27 = 0x27,
-    SONG_28 = 0x28,
-    SONG_29 = 0x29,
-    SONG_2A = 0x2A,
-    SONG_2B = 0x2B,
-    SONG_2C = 0x2C,
-    SONG_BGM_EVT_05 = 0x02F, // "Lights in the Dark"
-    SONG_31 = 0x31,
-    SONG_32 = 0x32,
-    SONG_33 = 0x33,
-    SONG_BGM_SELECT = 0x034, // "Combat Preparation"
-    SONG_37 = 0x37,
-    SONG_3B = 0x3B,
-    SONG_3C = 0x3C,
-    SONG_3D = 0x3D,
-    SONG_3E = 0x3E,
-    SONG_3F = 0x3F,
-    SONG_43 = 0x43,
-    SONG_44 = 0x44,
-    SONG_45 = 0x45,
-    SONG_BGM_ED_AFTER = 0x046,
-    SONG_BGM_ED_STAFF_2 = 0x056,
+    SONG_NONE = 0x00,
+    SONG_THE_VALIANT = 0x01,
+    SONG_MAIN_THEME = 0x02,
+    SONG_PROLOGUE = 0x03,
+    SONG_THE_BEGINNING = 0x004,
+    SONG_TREASURED_MEMORIES = 0x005,
+    SONG_GRIM_JOURNEY = 0x006,
+    SONG_TIES_OF_FRIENDSHIP = 0x007,
+    SONG_RAY_OF_HOPE = 0x008,
+    SONG_DISTANT_ROADS = 0x09,
+    SONG_RISE_ABOVE = 0x0A,
+    SONG_FOLLOW_ME = 0x0B,
+    SONG_DETERMINATION = 0x0C,
+    SONG_TRUTH_DESPAIR_AND_HOPE = 0x0D,
+    SONG_LAND_OF_PROMISE = 0x0E,
+    SONG_BINDING_VOW = 0x0F,
+    SONG_GRASP_AT_VICTORY = 0x10,
+    SONG_SHADOWS_APPROACH = 0x11,
+    SONG_CONFRONT_THE_PAST = 0x12,
+    SONG_SHADOW_OF_THE_ENEMY = 0x13,
+    SONG_ASSAULT = 0x14,
+    SONG_FROM_THE_DARKNESS = 0x15,
+    SONG_TWISTED_SHADOWS_A = 0x16,
+    SONG_TWISTED_SHADOWS_B = 0x17,
+    SONG_ENVOY_FROM_THE_DARK = 0x18,
+    SONG_ATTACK	= 0x19,
+    SONG_DEFENSE = 0x1A,
+    SONG_THE_BATTLE_MUST_BE_WON	= 0x1B,
+    SONG_POWERFUL_FOE =	0x1C,
+    SONG_THE_PRINCES_DESPAIR =	0x1D,
+    SONG_RETURN_OF_THE_DEMON_KING =	0x1E,
+    SONG_SACRED_STRENGTH =	0x1F,
+    SONG_TETHYS	=	0x20,
+    SONG_HEALING =	0x21,
+    SONG_CURING = 0x22,
+    SONG_TO_A_HIGHER_PLACE = 0x23,
+    SONG_ADVANCE = 0x24,
+    SONG_TENSION = 0x25,
+    SONG_RAID = 0x26,
+    SONG_REUNION = 0x27,
+    SONG_INDIGNATION = 0x28,
+    SONG_SORROW = 0x29,
+    SONG_LAUGHTER = 0x2A,
+    SONG_LEGACY = 0x2B,
+    SONG_LYON = 0x2C,
+    SONG_LOST_HEART = 0x2D,
+    SONG_SOLVE_THE_RIDDLE = 0x2E,
+    SONG_LIGHTS_IN_THE_DARK = 0x2F,
+    SONG_COMRADES = 0x30,
+    SONG_VICTORY = 0x31,
+    SONG_INTO_THE_SHADOW_OF_VICTORY = 0x32,
+    SONG_MAIN_THEME_REPRISE = 0x33,
+    SONG_COMBAT_PREPARATION = 0x34,
+    SONG_SHOPS = 0x35,
+    SONG_ARMORIES = 0x36,
+    SONG_BONDS = 0x37,
+    SONG_COLOSSEUM_ENTRANCE = 0x38,
+    SONG_IN_THE_COLOSSEUM = 0x39,
+    SONG_COLOSSEUM_VICTORY = 0x3A, 
+    SONG_COLOSSEUM_DEFEAT = 0x3B,
+    SONG_VICTORY_SONG_A = 0x3C,
+    SONG_VICTORY_SONG_B = 0x3D,
+    SONG_GAME_OVER = 0x3E,
+    SONG_IN_SORROWS_SHROUD = 0x3F,
+    SONG_RECORDS = 0x40,
+    SONG_VICTORY_AND_THE_FUTURE = 0x41,
+    SONG_FLY_WITH_THE_BREEZE = 0x42,
+    SONG_43 = 0x43, //title ?
```

Comment:
`0x43` could be `SONG_TITLE_SCREEN` or `SONG_MAIN_THEME` since it is the clipped version of the title screen theme  that plays when you skip the intro cutscene